src/main/kotlin/com/devfloor/untitled/common/exception/NotFoundException.kt

```diff
@@ -0,0 +1,6 @@
+package com.devfloor.untitled.common.exception
+
+data class EntityNotFoundException(
+    override val message: String
+) : RuntimeException() {
+}
```

Comment:
```suggestion
data class EntityNotFoundException(override val message: String) : RuntimeException()
```

이렇게 한 줄로 처리 가능할 듯? 그리고 파일 이름이랑 클래스 이름이랑 달라! 파일 이름을 클래스 이름에 맞추면 될 듯!